### DISCUSSION TITLE
Welcome to shortest Discussions!

### DISCUSSION THREAD

**Discussion by slavingia** [antiwork/shortest#38] [Announcements]
Going to discuss our roadmap and help people ideate/scope potential Shortest improvements with whomever wants to join!

Join our first meeting here: https://calendar.app.google/TvtPaYWgrUpd8xM18

Monday, October 7 · 11:00am – 12:00pm
Time zone: EST (New York)

**Comment by crabest** [comment]
it's gonna be recorded right?

**Reply by slavingia** [reply]
Sure!

**Comment by slavingia** [comment]
Call here:
https://youtu.be/SIUmFbMOchM

**Comment by slavingia** [comment]
A good small one for someone to tackle: https://github.com/gumroad/shortest/issues/33
